### DISCUSSION TITLE
How to exclude a specific remote folder to be pruned?

### DISCUSSION THREAD

**Discussion by pardub** [borgbase/vorta#1726] [FAQ] [answered]
Hi,
I have a folder on my laptop that I want to backup on a remote server.
I want to delete this folder locally from my laptop once it has been uploaded but I want  to keep it permanently on the remote server and never be deleted. That folder won't be updated.
I didn't see anything in the pruning options to take action for a specific folder
Is it possible to achieve this?
Thanks

**Comment by real-yfprojects** [comment, marked as answer]
You could rename the archive so that it doesn't start with the prune-prefix you configured.

**Reply by pardub** [reply]
Nice workaround! I didn't think about that!Thank you!!
